DependencyInjection/DoctrineExtension.php

```diff
@@ -193,9 +195,20 @@ protected function dbalLoad(array $config, ContainerBuilder $container)
         $container->setParameter('doctrine.connections', $connections);
         $container->setParameter('doctrine.default_connection', $this->defaultConnection);
 
+        $idleConnectionAction = true;
+
+        if (! class_exists(IdleConnectionMiddleware::class) || ! class_exists(Listener::class)) {
```

Comment:
It'd move this "if" to the end of the method so that we don't need the $idleConnectionAction variable
```suggestion
        if (! class_exists(Listener::class)) {
```